plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -2005,13 +2047,15 @@ private static SqlVarbinary byteArrayToVarbinary(byte[] input)
         return new SqlVarbinary(input);
     }
 
-    private static Timestamp intToTimestamp(Integer input)
+    private static Timestamp intToTimestamp(int inputMillis)
     {
-        if (input == null) {
-            return null;
-        }
-        long seconds = (input / 1000);
-        int nanos = ((input % 1000) * 1_000_000);
+        return longToTimestamp(toIntExact(inputMillis));
+    }
+
+    private static Timestamp longToTimestamp(long inputMillis)
+    {
+        long seconds = (inputMillis / 1000);
+        int nanos = Math.floorMod(inputMillis, 1000) * 1_000_000;
```

Comment:
- use floorDiv in the first line
- import floorMod statically
- since you're using floorMod now, the code `if (nanos < 0)` became dead code, right?
- when changing this, use named constants: MILLIS_PER_S..., NANOS_PER_MS...
- extract this refactor to a separate commit

Reply:
Implemented in commit `Simplify implementation of AbstractTestParquetReader#intToTimestamp`